### PR TITLE
Create continuous_integration.yml

### PR DESCRIPTION
**Pull request by suricactus**

### CODE REVIEW

File: .github/workflows/continuous_integration.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        run: |
+          make gettext
+          tx push -s
+
+      - name: Translations push
+        run: |
+          set -e
+
+          make clean
+          if ${GITHUB_REF##*/} = "master";
```
Comment (m-kuhn) [inline .github/workflows/continuous_integration.yml:55]: is this the right condition (the *secure* master branch term below)?

Comment (suricactus) [reply .github/workflows/continuous_integration.yml:55]: Copy paste from `travis.yml`

Comment (suricactus) [reply .github/workflows/continuous_integration.yml:55]: I guess we only want to run transifex on master, right? Then I think this is the right condition and the message needs to be updated?

Comment (m-kuhn) [reply .github/workflows/continuous_integration.yml:55]: Yes, but the condition is for "secrets" related code, so I guess it shouldn't only be triggered by master?

Comment (suricactus) [reply .github/workflows/continuous_integration.yml]: After second thought I think there is no need for the if condition at all. We can trigger the build, only we should filter whether the changes are commited to QField docs website.

File: .github/workflows/continuous_integration.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+          tlmgr install babel-romanian
+          tlmgr install babel-italian
+          tlmgr install babel-spanish
+
+      - name: Build docs
+        run: |
+          set -e
+
+          make gettext
+          tx push -s
```
Comment (m-kuhn) [inline .github/workflows/continuous_integration.yml:43]: Only on master

**Review by m-kuhn — APPROVED** [review]

**Comment by suricactus** [comment]
Amen!

**Pull request merged**